.github/workflows/ci_tests.yml

```
@@ -33,6 +33,9 @@ jobs:
           - os: ubuntu-latest
             python: '3.10'
             tox_env: 'py310-test-alldeps'
+          - os: ubuntu-latest
+            python: '3.11.0-rc.2'
+            tox_env: 'py311-test-alldeps'
```

It won't pass with stable astropy (yet).

```suggestion
            tox_env: 'py311-test-devdeps'
```

---

thanks